### ISSUE TITLE
Custom tabs do not survive a restart

### ISSUE THREAD

**Issue by Retzudo** [meetfranz/plugins#22]
Looks like issue #10 has been re-introduced with Franz 4.0.3

Steps to reproduce:

1. I download [Franz 4.0.3 (Linux, 64-bit)](https://github.com/meetfranz/franz-app/releases/download/4.0.3/Franz-linux-x64-4.0.3.tgz) and extract it somewhere.
2. I open Franz and see my existing services
3. I add a new either a pre-installed (e.g. Skype) or custom service. It then shows up as a new tab.
4. I close Franz with File → Quit.
5. I re-open Franz 4.0.3. The tab just created is now missing.

I get this error right after starting Franz which seems unrelated to this issue:

```
TypeError: Error processing argument at index 0, conversion failure from /home/chris/franz/resources/app.asar/assets/tray/linux/franz-taskbar@2x.png
    at TypeError (native)
    at createSystemTray (/home/chris/franz/resources/app.asar/lib/tray.js:1:1254)
    at EventEmitter.<anonymous> (/home/chris/franz/resources/app.asar/main.js:1:14138)
    at emitOne (events.js:101:20)
    at EventEmitter.emit (events.js:188:7)
```

**Comment by adlk** [comment]
Hey @Retzudo we had a minor git issue. We will publish a new update this afternoon.

**Comment by Retzudo** [comment]
Thanks, solved with Franz 4.0.4.
